Libraries/Components/TextInput/TextInput.js

```diff
@@ -1146,7 +1146,6 @@ const TextInput = createReactClass({
         rejectResponderTermination={true}
         accessible={props.accessible}
         accessibilityLabel={props.accessibilityLabel}
-        accessibilityHint={props.accessibilityHint} // TODO(OSS Candidate ISS#2710739)
```

Comment:
We use this prop in the macOS fork.  Please restore.   The comment can be changed to `TODO(macOS ISS#2323203)`